frontend/src/components/NoMatch.tsx

```diff
@@ -0,0 +1,47 @@
+import {Box, Link} from '@cloudscape-design/components'
+import React from 'react'
+import TopBar from './TopBar'
+import errorPage from './../../public/img/error_pages_illustration.svg'
```

Comment:
issue: please make sure this illustration is covered by our licenses ([see for instance here](https://github.com/aws-samples/pcluster-manager/blob/main/frontend/public/flags/Flags.LICENSE))